src/Symfony/Component/VarDumper/Caster/AddressInfoCaster.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @final
+ */
+class AddressInfoCaster
+{
+    private const MAPS = [
+        'ai_flags' => [
+            1 => 'AI_PASSIVE',
+            2 => 'AI_CANONNAME',
+            4 => 'AI_NUMERICHOST',
+            8 => 'AI_V4MAPPED',
+            16 => 'AI_ALL',
+            32 => 'AI_ADDRCONFIG',
+            64 => 'AI_IDN',
+            128 => 'AI_CANONIDN',
+            1024 => 'AI_NUMERICSERV',
+        ],
+        'ai_family' => [
+            1 => 'AF_UNIX',
+            2 => 'AF_INET',
+            10 => 'AF_INET6',
```

Comment:
The PHP documentation also has a `AF_DIVERT` constant: https://www.php.net/manual/en/sockets.constants.php#constant.af-divert

Reply:
updated, with constant value resolution when possible (to override the hardcoded value since it can change depending on the OS)